Front-End/challenge2/script.js

```diff
@@ -5,3 +5,52 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+const btn_number = document.getElementsByTagName("button")
```

Comment:
Variable names in JS are camelCase

Reply:
I also myself like to avoid abbreviations when naming variables.

`const buttonNumber = ...`